### ISSUE TITLE
Is this JSON support via RedisJSON from redislabs available in this client?

### ISSUE THREAD

**Issue by xtianus79** [redis/ioredis#1031]
If not how would you use raw queries to access the JSON module loaded?

**Comment by xtianus79** [comment]
this isn't stale but the answer is to use send_command()

**Comment by xtianus79** [comment]
Why was this closed?

**Comment by luin** [comment]
Sorry for the late response. Is this issue duplicated with https://github.com/luin/ioredis/issues/525?

**Comment by xtianus79** [comment]
@luin Yes it is a bit of a duplicate but If anything there should be an update to the documentation about the redis.call() support. And, a lot of functionality is lost such as typing if that is not added.

**Comment by PavelPashov** [comment]
Closing this as there's an existing example of using the JSON module [here](https://github.com/redis/ioredis/blob/main/examples/module.js)
